.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,27 +1,5 @@
-<!--
-###############################################################
-# Copyright (c) 2024 Contributors to the Eclipse Foundation
-#
-# See the NOTICE file(s) distributed with this work for additional
-# information regarding copyright ownership.
-#
-# This program and the accompanying materials are made available under the
-# terms of the Apache License, Version 2.0 which is available at
-# https://www.apache.org/licenses/LICENSE-2.0.
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-#
-# SPDX-License-Identifier: Apache-2.0
-###############################################################
--->
-
```

Comment:
why did you remove the header again?

Reply:
Added: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/69c58f27521831fc749979f9248d4ae836418a7e